k9mail-library/src/main/java/com/fsck/k9/mail/store/webdav/WebDavStore.java

```diff
@@ -307,50 +332,44 @@ public boolean isCopyCapable() {
     }
 
     private String getSpecialFoldersList() {
-        StringBuilder builder = new StringBuilder(200);
-        builder.append("<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>");
-        builder.append("<propfind xmlns=\"DAV:\">");
-        builder.append("<prop>");
-        builder.append("<").append(WebDavConstants.DAV_MAIL_INBOX_FOLDER).append(" xmlns=\"urn:schemas:httpmail:\"/>");
-        builder.append("<").append(WebDavConstants.DAV_MAIL_DRAFTS_FOLDER).append(" xmlns=\"urn:schemas:httpmail:\"/>");
-        builder.append("<").append(WebDavConstants.DAV_MAIL_OUTBOX_FOLDER).append(" xmlns=\"urn:schemas:httpmail:\"/>");
-        builder.append("<").append(WebDavConstants.DAV_MAIL_SENT_FOLDER).append(" xmlns=\"urn:schemas:httpmail:\"/>");
-        builder.append("<").append(WebDavConstants.DAV_MAIL_TRASH_FOLDER).append(" xmlns=\"urn:schemas:httpmail:\"/>");
+        String builder = "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>" +
```

Comment:
The variable name should now probably be something like `specialFoldersListRequest`.